docs/changes/2480.feature.rst

```diff
@@ -0,0 +1,3 @@
+The muon ring fitter needs to be improved.
```

Comment:
This is a description of an issue, not an implemented change. Please describe what you _changed_ in this pull request.

Also, the pull request number is 2736, not 2480.